gen/gen.go

```diff
@@ -254,19 +255,25 @@ var SwaggerInfo = swaggerInfo{
 type s struct{}
 
 func (s *s) ReadDoc() string {
+	sInfo := SwaggerInfo
+	sInfo.Description = parseAndExecuteTemplate(sInfo.Description, sInfo)
+	return parseAndExecuteTemplate(doc, sInfo)
+}
+
+func parseAndExecuteTemplate(text string, info swaggerInfo) string {
 	t, err := template.New("swagger_info").Funcs(template.FuncMap{
 		"marshal": func(v interface{}) string {
 			a, _ := json.Marshal(v)
 			return string(a)
 		},
-	}).Parse(doc)
+	}).Parse(text)
 	if err != nil {
-		return doc
+		return text
```

Comment:
Why are u ignoring errors ?

Reply:
The errors are ignored today, that is nothing I changed.